package.json

```diff
@@ -152,6 +152,7 @@
     "core-js": "^3.20.1",
     "crypto-browserify": "^3.12.0",
     "d3": "^6.7.0",
+    "d3-color": "3.1.0",
```

Comment:
Was this because we haven't built in a while?

Reply:
Yes, there was an audit finding.